lib/iris/tests/unit/fileformats/netcdf/test_Saver.py

```diff
@@ -472,6 +472,33 @@ def test_masked_byte_fill_value_passed(self):
                 pass
 
 
+class Test_cf_valid_var_name(tests.IrisTest):
+    def test_no_replacement(self):
+        self.assertEqual(Saver.cf_valid_var_name('valid_Nam3'),
+                         'valid_Nam3')
+
+    def test_special_chars(self):
+        self.assertEqual(Saver.cf_valid_var_name('inv?alid'),
+                         'inv_alid')
+
+    def test_leading_underscore(self):
+        self.assertEqual(Saver.cf_valid_var_name('_invalid'),
+                         'var__invalid')
+
+    def test_leading_number(self):
+        self.assertEqual(Saver.cf_valid_var_name('_invalid'),
```

Comment:
Copy & paste fail?

`_invalid` doesn't have a leading number... Anyways, the first character may be alphanumeric.

Reply:
👍